.github/workflows/moodle-ci.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   test:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
```

Comment:
FYI: CI was failing due to the out-dated ubuntu version. c.f. https://stackoverflow.com/questions/70959954/error-waiting-for-a-runner-to-pick-up-this-job-using-github-actions

Reply:
Thanks, Kei